dune_client/viz/graphs.py

```diff
@@ -0,0 +1,88 @@
+"""
+Functions you can call to make different graphs
+"""
+
+from typing import Dict, Union
+
+# https://github.com/plotly/colorlover/issues/35
+import colorlover as cl  # type: ignore[import]
+import pandas as pd
+import plotly.graph_objects as go  # type: ignore[import]
+from plotly.graph_objs import Figure  # type: ignore[import]
+
+
+# function to create Sankey diagram
+def create_sankey(
+    query_result: pd.DataFrame,
+    predefined_colors: Dict[str, str],
+    columns: Dict[str, str],
+    viz_config: Dict[str, Union[int, float]],
+    title: str = "unnamed",
+) -> Figure:
+    """
+    Creates a Sankey diagram based on input query_result
+    , which must contain source, target, value columns
```

Comment:
Nit: Something about having a comma at the start of a line makes me feel uneasy
```suggestion
    Creates a Sankey diagram based on input query_result,
    which must contain source, target, value columns
```